lint/src/test/java/com/stripe/android/lint/ComposeCollectAsStateUsageDetectorTest.kt

```diff
@@ -0,0 +1,101 @@
+package com.stripe.android.lint
+
+import com.android.tools.lint.checks.infrastructure.LintDetectorTest.kotlin
+import com.android.tools.lint.checks.infrastructure.TestLintTask.lint
+import org.junit.Test
+
+class ComposeCollectAsStateUsageDetectorTest {
```

Comment:
What if I use it without an import statement? (Can I even do that? idk)

Reply:
It isn't possible to directly import it but you can wildcard import.

```kotlin
import androidx.compose.runtime.*
....
stateFlow.collectAsState()
```

Reply:
Would our lint rules block the wildcard import?

Reply:
I'll look into detecting it this way as well.

Reply:
Talked in Slack, we catch wildcard imports so this should be fine.